utils/quick_setup.sh

```diff
@@ -44,7 +44,7 @@ fi
 # if an install is already present, remove it to start from a clean slate
 rm -rf ironenv
 rm -rf my_install
-python3.10 -m venv ironenv
+python3 -m venv ironenv
```

Comment:
Maybe this should still enforce python3.10 or python3.12, just in case users have those in their environment satisfying the check but they aren't the default

Reply:
Agreed. That would be clear and safe